### ISSUE TITLE
Display another server's timeline in a column in advanced interface

### ISSUE THREAD

**Issue by eobet** [mastodon/mastodon#19307]
### Pitch

Some servers are really focused on more or less one interest, sort of like a giant hashtag, so it would be nice if you could put the entire server's timeline in a column in the advanced web interface.

### Motivation

Coming from a different social media platform, having to choose a server is a huge hurdle and Mastodon can still feel "isolating". Having multiple accounts is a hassle and you can't take your content with you when you migrate, so would it not be nice to at least be able to listen in on what's going on in another server?

**Comment by Cassolotl** [comment]
I like this idea a lot.

Related to *[FEATURE] Follow public local timelines from other instances #6942*.

**Comment by brandonrobertz** [comment]
Yeah this was the first thing I tried to do when I used mastodon and quickly found that it's not possible. I wish it was possible to "explore" only a specific server's content.

**Comment by robkam** [comment]
The Qoto instance already does this ([repository here](https://git.qoto.org/qoto/qoto)):

- Domain Subscriptions - Bring another instance's local timeline as a feed in Qoto, no need to have an account on another server again, just import their timeline here!
- Favorite Domains - Our last release allowed us to add remote timeline as a feature where you could pull up the local timeline of a remote instance as its own column. Now you can also add domains to a favorite domain list which makes it easier to access and bring up the timelines by listing them directly in the navigation panel.

**Comment by vitorgrs** [comment]
Even on the standard interface and mobile app it should be a priority IMO. Also searching a subject on a specific instance. 
The main problem of Mastodon is discoverability of users/talks. And the main issue it's because users group on specific instances, and a lot of these instances are closed...

**Comment by rriemann** [comment]
How about this?

1. Instead of searching for @user@other_instance, I could search for @other_instance to access all users my instance known from @other_instance. An extra link could be displayed to see all of their posts known to my instance
2. unless other_instance is an activitypub-enabled domain, it is treated as a query for a user
3. Would be nice to autocomplete attempts to search for other_instance while typing the search

**Comment by eusebiu** [comment]
> Even on the standard interface and mobile app it should be a priority IMO. Also searching a subject on a specific instance. The main problem of Mastodon is discoverability of users/talks. And the main issue it's because users group on specific instances, and a lot of these instances are closed...

Yes, exactly! This will make users easily build their timeline with exactly their preferences!
Seeing an empty timeline in Home is not the best UX.
And this should be in the standard interface...

**Comment by mthomason12** [comment]
Came here to post exactly this suggestion, glad I wasn't the only one thinking about it!

Use case:
I'm on warhammer.social
Other servers relevant to my interests are dice.camp, tabletop.social, and chirp.enworld.org.  Many people I already know may even have joined these but there's no way for me to find out without asking each person individually.

I'd like posts from those servers to show up somewhere (either in a new panel or mixed into my local timeline if I request it, I don't really mind where) to help me find people to follow from the wider community.
